web/styles/popovers.css

```diff
@@ -347,6 +347,36 @@ ul {
         outline: 10px solid hsl(0deg 0% 100%);
     }
 
+    .popover-avatar-background {
+        background-color: hsl(0deg 0% 0%);
+        height: 100%;
+        width: 100%;
+        opacity: 0.6;
+        display: none;
+    }
+
+    .exit_user_info_popover {
+        color: hsl(0deg 0% 100%);
+        cursor: pointer;
+        display: none;
+        position: absolute;
+        font-size: 2rem;
+        top: 0;
+        right: 10px;
+    }
+
+    &:hover,
+    &:active {
+        .popover-avatar-background {
+            display: block;
+        }
+
+        .exit_user_info_popover {
+            display: block;
+            opacity: 1;
+        }
+    }
+
```

Comment:
We need to have this CSS because we want to have an overlay effect on the image when hovered so that the close button can appear. Without the `popover-avatar-background` div, it is not possible to implement the overlay effect. Additionally, we do not want the close button to appear all the time on the popover; it should only appear when the image is hovered.